### ISSUE TITLE
Lua replacement

### ISSUE THREAD

**Issue by Imposter** [IndigoGames/IndigoHAWX-Client#6]
Create separate Lua state and manage that instead of the game's states. Call InstallGameFunctions on the Lua state so we can access the game's functions.

**Comment by Nukem9** [comment]
Why keep the states separate, wouldn't there be multithreading problems?

**Comment by Imposter** [comment]
Not really, we'll only be using the game's functions for extending user scripts. That shouldn't interfere with the game whatsoever.
